mythril/laser/ethereum/instructions.py

```diff
@@ -1609,7 +1571,31 @@ def delegatecall_post(self, global_state: GlobalState) -> List[GlobalState]:
     def staticcall_(self, global_state: GlobalState) -> List[GlobalState]:
         # TODO: implement me
```

Comment:
can you remove this?

Reply:
It's not implemented. Only natives are implemented.